dashboard/test/lib/pd/survey_pipeline/retriever_test.rb

```diff
@@ -0,0 +1,93 @@
+require 'test_helper'
+require 'pd/survey_pipeline/retriever.rb'
+
+module Pd::SurveyPipeline
+  class DailySurveyRetrieverTest < ActiveSupport::TestCase
+    include Pd::WorkshopConstants
+
+    setup_all do
+      @form_ids = [11_000_000_000_000, 11_000_000_000_001]
```

Comment:
is `@form_ids` specifically workshop daily survey form ids? since this is not a superset of all form ids, can this variable have a more descriptive name?

Reply:
Will change to `workshop_form_ids`